spec/jobs/subscription_confirm_job_spec.rb

```diff
@@ -190,23 +190,6 @@
           job.send(:confirm_order!, order)
         end
       end
-
-      context "Stripe Connect" do
```

Comment:
We should actually keep these specs. Luis updated them to use the SCA payment method and just forgot to change the `context` line.

Reply:
correct :-)

can be changed to "Stripe SCA"